packages/app/src/cli/models/extensions/specifications/marketing_activity_schemas/marketing_activity_schema.test.ts

```diff
@@ -53,15 +52,7 @@ describe('MarketingActivityExtensionSchema', () => {
           ...config,
           fields: ['not an object'],
         }),
-      ).toThrowError(
-        new zod.ZodError([
-          {
-            message: 'Field must be an object',
-            code: zod.ZodIssueCode.custom,
-            path: ['fields', 0],
-          },
-        ]),
-      )
+      ).toThrow('Field must be an object')
```

Comment:
Why are these errors failing?

> Vitest now checks more properties when comparing errors via `toEqual` or `toThrowError`. Vitest now compares `name`, `message`, `cause` and `AggregateError.errors`

Is there something here that ends up different? Would be good to understand for future reference.